layers/timeline_semaphore.c

```diff
@@ -1253,7 +1253,7 @@ maybe_clone_semaphore(struct queue_data *queue,
     wait_point->point = NULL;
     wait_point->semaphore->queue = queue;
 
-    VkPipelineStageFlags wait_stage_mask = 0;
+    VkPipelineStageFlags wait_stage_mask = VK_PIPELINE_STAGE_BOTTOM_OF_PIPE_BIT;
```

Comment:
I wonder if we shouldn't instead use the stage specified by the caller of `vkQueueSubmit()` and use `BOTTOM_OF_PIPE_BIT` for `vkQueueBindSparse`

Reply:
Does it matter at all which stage is specified here given that there are no command buffers in this submission?

Reply:
I think you're right, thanks.